src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerActionInvokerCache.cs

```diff
@@ -56,22 +56,28 @@ public Entry GetCacheEntry(ControllerContext controllerContext)
                 return entry;
```

Comment:
`GetFilters` and `GetControllerActionMethodExecutor` should we deleted. We do not want to look up the cache entry more than once per request.